### ISSUE TITLE
cat-select a11y with screenreaders

### ISSUE THREAD

**Issue by JonayLo** [haiilo/catalyst#415]
When screenreader is on and you first navigate to a cat-select using tab, the dropdown menu is displayed but is not accessible using arrow keys
